docs/plio-for-teams/single-sign-on.md

```diff
@@ -30,4 +30,22 @@ You need to share this link with your viewers instead of the usual plio link and
 
 There are two ways to use this feature:
 - If you already have a user management system, you can simply pass in the unique ID for a viewer in your system as the `unique_id` parameter mentioned above.
-- If you don't and simply want any viewer to bypass Plio's authentication, you can use any random value for `unique_id`. However, in this case, you won't be able to identify the users during analysis. 
\ No newline at end of file
+- If you don't and simply want any viewer to bypass Plio's authentication, you can use any random value for `unique_id`. However, in this case, you won't be able to identify the users during analysis. 
+
+### Receiving data from Embedded Plio
+
+You can embed your plios on your website as shown [here](../getting-started/#embedding-a-plio) but, on the default plan, you will not receive any data from the embedded plio if your viewers haven't already logged in to plio. 
+
+However, if you have an organizational workspace, you can receive data from your users without them having to log in to plio using Single-Sign On (SSO). Let's see how!
```

Comment:
```suggestion
However, if you have an organizational workspace, you can use the Single-Sign On (SSO) feature provided by Plio. This will allow you to receive data from your users without them having to log in to plio. Let's see how!
```
Just trying to break long sentences into multiple senteces